charts/incubator/tileserver-gl/values.yaml

```diff
@@ -0,0 +1,25 @@
+image:
+  repository: maptiler/tileserver-gl
+  tag: v4.3.3
+  pullPolicy: IfNotPresent
+
+args:
+  - -b
+  - "0.0.0.0"
+  - -p
+  - "{{ .Values.service.main.ports.main.port }}"
+
```

Comment:
Don't need these extra args, the ports are set below

Reply:
When an app offer the ability to change the internal listening port, we like to tie it with the external port.
So there is one less port to configure.

Also some apps like to have both ports same, (eg qbit :P)